dataservice/api/participant/models.py

```diff
@@ -14,3 +14,5 @@ class Participant(db.Model, Base):
     """
     __tablename__ = "participant"
     external_id = db.Column(db.String(32))
+    samples = db.relationship('Sample', backref='participant',
```

Comment:
Sorry I didn't think of this before, but I think you may want to add the `lazy="joined"` parameter to db.relationship here. I did this for Diagnosis and I think we will want the same for Sample. See explanation [here](https://github.com/kids-first/kf-api-dataservice/pull/67#discussion_r163430583)